### ISSUE TITLE
Comskip issues

### ISSUE THREAD

**Issue by rdagijones** [Unmanic/unmanic#352]
I have installed Unmanic on Unraid, added the Comskip plugin and set up one Worker to run Comskip. When I drop a file in the monitored directory to test it, the task completes "successfully" in about 4 seconds, but I get the following error:

> Runner did not request to execute a command

> No Plugin requested to run commands for this file '/library/tv/Green Acres/Season 3/Green Acres S03E07 Dont Count Your Tomatoes Before Theyre Picked3.ts'

Also, no edl file is created and the video file is unchanged at all.

![ComskipPluginConfigurations](https://user-images.githubusercontent.com/5282048/174194330-f6acb022-321c-4615-b13f-ee263ea9d526.jpg)

**Comment by Josh5** [comment]
Did you restart the container after installing the plugin?

**Comment by rdagijones** [comment]
Yes, I have restarted Unmanic more than once since installing the Comskip plugin and yet I still get the same behavior.

**Comment by lukepolo** [comment]
I all the sudden started to get this as well.

**Comment by Josh5** [comment]
@lukepolo This is not necessarily an issue. And is almost definitely not the same thing as was being described in this comskip issue

**Comment by Josh5** [comment]
@rdagijones I just realised that this could be caused by a missing mimetype on the system that you are using. It may not recognise ts as a video file format. 

I will double check if the docker image is setting ts as a video mimetype

**Comment by Josh5** [comment]
Never mind, I just checked and ts is definitely configured in this docker image

**Comment by lukepolo** [comment]
My issue was a permission issue

**Comment by iankendrick** [comment]
I'm seeing the same issue as @rdagijone . I don't believe its a permission issue because Unmanic is able to transcode files within the same Task. The comskip plugin step just reports "Runner did not request to execute a command". This happens no matter how I have Comskip plugin configured. I am using Docker and have restarted the container after installing the plugin.

<img width="994" height="258" alt="Image" src="https://github.com/user-attachments/assets/3bc14b09-7721-4f5c-b8f7-bc31349cf8d4" />
